### PR TITLE
Add type freezing paper.

### PR DESCRIPTION
**Pull request by Pavel-Durov**

### CODE REVIEW

File: softdev.bib [hunk 1]
```diff
@@ -3333,3 +3333,10 @@ @article{yanovski21ghostcell
   year={2021},
   month = Aug,
 }
+
+@misc{cheng20typefreezing,
+    title = "Type freezing: exploiting attribute type monomorphism in tracing JIT compilers",
+    author = "Cheng, Lin and Ilbeyi, Berkin and Bolz-Tereick, Carl Friedrich and Batten, Christopher",
+    year = 2020,
+    isbn = {9781450370479},
```
Comment (Pavel-Durov) [inline softdev.bib:3341]: I that sufficient? or does it need more info?

File: softdev.bib [hunk 2]
```diff
@@ -3333,3 +3333,10 @@ @article{yanovski21ghostcell
   year={2021},
   month = Aug,
 }
+
+@misc{cheng20typefreezing,
+    title = "Type freezing: exploiting attribute type monomorphism in tracing JIT compilers",
```
Comment (ltratt) [inline softdev.bib:3338]: You'll need to put curly braces around "JIT" or it'll be lower-cased.

File: softdev.bib (hunk 1 above)
Comment (ltratt) [reply softdev.bib:3341]: What event was this published at? Please see other entries in the file to get a sense of the possibilities.

File: softdev.bib [hunk 3]
```diff
@@ -3333,3 +3333,10 @@ @article{yanovski21ghostcell
...
+    author = "Cheng, Lin and Ilbeyi, Berkin and Bolz-Tereick, Carl Friedrich and Batten, Christopher",
+    year = 2020,
```
Comment (ltratt) [inline softdev.bib:3340]: We also ask for a month if at all possible.

Comment (Pavel-Durov) [reply softdev.bib:3340]: Month added :point_right: 3886dfe9e36eb83a454e9f3ecd7b39135f8191d5

File: softdev.bib (hunk 2 above)
Comment (Pavel-Durov) [reply softdev.bib]: added :point_right: b6f38e5d98b984a25b021e9e64a98b5c8a568c88

File: softdev.bib (hunk 1 above)
Comment (Pavel-Durov) [reply softdev.bib]: Is the event a publisher?
Added publisher :point_right: 98eb9ccdd96826c197611bd86e96311a75c06d25

Comment (ltratt) [reply softdev.bib]: The publisher isn't relevant to us. This was published at a conference or workshop, probably? If you look at other entries you'll see how to deal with such a case.

Comment (Pavel-Durov) [reply softdev.bib]: Looking at the other entries, I guess it should be set as `booktitle`, I didn't realise that it's used for events.
Updated :point_right: 2d59f6f340598a8bc335ff2f764c648e694a45b6

Comment (ltratt) [reply softdev.bib]: That's reasonable but note you now need to make the entry `inproceedings`.

Comment (Pavel-Durov) [reply softdev.bib]: ok. done :point_right: 118d3fc7fad24b55cc5aadaacd0ddce7a2408f6e

**Comment by ltratt** [comment]
Please squash.

**Comment by Pavel-Durov** [comment]
Squashed :point_right: 728420e45f4ea15a5602d1033092865b08d08409

**Pull request merged**